pkg/util/log/structured.go

```diff
@@ -148,7 +148,7 @@ func addStructured(ctx context.Context, s Severity, depth int, format string, ar
 			reportable = fmt.Sprintf("%T", args[0])
 		}
 		reportable = fmt.Sprintf("%s:%d %s", filepath.Base(file), line, reportable)
-		sendCrashReport(ctx, reportable, depth+1)
+		sendCrashReport(ctx, nil /* no ReportingSettings available */, reportable, depth+1)
```

Comment:
I'm not thrilled about this. Since we can't plumb this one, I'd rather see a log.SendCrashReports bool that gets updated using an OnChange callback in the setting in the registry.